const f = () => throw Error('hi');
